account_bank_statement_import_qif/account_bank_statement_import_qif.py

```diff
@@ -6,41 +6,49 @@
 import StringIO
 
 from openerp.tools.translate import _
-from openerp.osv import osv, fields
+from openerp import api, models, fields
 from openerp.exceptions import Warning
 
-class account_bank_statement_import(osv.TransientModel):
-    _inherit = "account.bank.statement.import"
 
-    _columns = {
-        'journal_id': fields.many2one('account.journal', string='Journal', help='Accounting journal related to the bank statement you\'re importing. It has be be manually chosen for statement formats which doesn\'t allow automatic journal detection (QIF for example).'),
-        'hide_journal_field': fields.boolean('Hide the journal field in the view'),
-    }
+class account_bank_statement_import(models.TransientModel):
+    _inherit = "account.bank.statement.import"
 
-    def _get_hide_journal_field(self, cr, uid, context=None):
-        return context and 'journal_id' in context or False
+    @api.model
+    def _get_hide_journal_field(self):
+        return self.env.context.get('journal_id') and True
 
-    _defaults = {
-        'hide_journal_field': _get_hide_journal_field,
-    }
+    journal_id = fields.Many2one(
```

Comment:
Please put this field in the main module, because there are other parsers that can use it (spanish one, for example), and it makes no sense to duplicate it when you have all installed.